PostReleaseActivities.md

```diff
@@ -0,0 +1,26 @@
+﻿Post-release activities
+=================================================================
+
+Please follow the below steps after publishing analyzer NuGet packages from this repo onto NuGet.org:
+
+1. Create a new release:
+   1. Draft: Click [here](https://github.com/dotnet/roslyn-analyzers/releases/new) to draft a new release. For reference, you can look at any of the existing releases, say [v2.9.3](https://github.com/dotnet/roslyn-analyzers/releases/edit/v2.9.3).
+   2. Release notes: Follow the steps in the *Steps to generate Release Notes* below to generate Release notes and copy the generated notes to the description section of the new release.
```

Comment:
Assuming we were updating a draft release for a release release with fixes / changes, we'd want to use the existing draft, right?

For prereleases, I think it makes sense to create a new release every time.

Reply:
Sure, that sounds reasonable too. Let me update the text accordingly.